tasks/main.yml

```diff
@@ -67,6 +67,24 @@
         state: touch
       when: ansible_distribution == "Fedora"
 
+- name: selinux
```

Comment:
The name of this block should be improved.  How about `Configure SELinux to allow freshclam downloads`?

Reply:
Resolved via commit f34d0bb.